crates/bevy_ui/src/ui_node.rs

```diff
@@ -26,6 +27,22 @@ impl Node {
     pub fn size(&self) -> Vec2 {
         self.calculated_size
     }
+
+    /// Returns the logical pixel coordinates of the UI node, based on its GlobalTransform.
```

Comment:
```suggestion
    /// Returns the logical pixel coordinates of the UI node, based on its `GlobalTransform`.
```

Reply:
Oh, of course, rewrote it too many times.